ext/standard/type.c

```diff
@@ -434,6 +450,135 @@ PHP_FUNCTION(is_callable)
 }
 /* }}} */
 
+#define RETURN_TYPE_NAME(t) { RETURN_STRINGL(t, sizeof(t) - 1) }
```

Comment:
minor; Create interned strings for these return values so that you're not creating a new string on every invocation.